Update publisher-verification-overview.md (#91852)
Azure/azure-content: articles/active-directory/develop/publisher-verification-overview.md

-  An MPN ID for a valid [Microsoft Partner Network](https://partner.microsoft.com/membership) account that has completed the [verification](/partner-center/verification-responses) process. This MPN account must be the [Partner global account (PGA)](/partner-center/account-structure#the-top-level-is-the-partner-global-account-pga) for your organization. 

> -  The Azure AD tenant where the app is registered must be associated with the Partner Global account. If it is not the primary tenant associated with the PGA, follow the steps [here] (/partner-center/multi-tenant-account#add-an-azure-ad-tenant-to-your-account) to set up the MPN parnter global account as a multi-tenant account and associate the Azure AD tenant 

Reviewer: ```suggestion
-  The Azure AD tenant where the app is registered must be associated with the Partner Global account. If it's not the primary tenant associated with the PGA, follow the steps to [set up the MPN partner global account as a multi-tenant account and associate the Azure AD tenant](/partner-center/multi-tenant-account#add-an-azure-ad-tenant-to-your-account). 
```
Fixed conjunction, link, link text, and typo
